src/helpers/downloadPdf.js

```diff
@@ -10,6 +10,7 @@ export default (dispatch, options = {}) => {
     filename = core.getDocument()?.getFilename() || 'document',
     includeAnnotations = true,
     externalURL,
+    useDisplayAuthor = false,
```

Comment:
Hmm I think we need to add some documentation for this new option in `apis/downloadPdf.js`.